README.md

```diff
@@ -23,10 +25,10 @@ Once you have `poetry` running, move to the next section.
 
     ```bash
     git clone git@github.com:Bitergia/trustable-grimoirelab-cli.git
```

Comment:
I think we should rename the repository too.

Reply:
I renamed the git clone to `git clone git@github.com:Bitergia/grimoirelab-metrics.git`, is that ok?

Reply:
Yes, that's ok.